roles/edpm_update/molecule/default/Containerfile.j2

```diff
@@ -6,6 +6,7 @@ FROM {{ item.image }}
 {% endif %}
 
 RUN curl -o /etc/yum.repos.d/delorean.repo https://trunk.rdoproject.org/centos9-wallaby/current/delorean.repo && \
+    curl -o /etc/yum.repos.d/delorean-deps.repo https://trunk.rdoproject.org/centos9-wallaby/delorean-deps.repo && \
```

Comment:
why wallaby deps repo needs to be added in Containerfile? no package seems to be using deps repo while building container. Tests installing antelope repos as needed.

Reply:
This is how the #623 set this up. @rebtoor maybe be better to answer this.

I added the additional repo because it contains the `centos<>-nfv-ovs` which contains all ovs packages. This is needed so when this test converges it finds latest ovs rpms when `dnf update openvswitch` is called

Reply:
yeap delorean.repo makes sense here for tripleo-repos rpm installation But not the deps repo. Also i see tripleo-repos also not used as using repo-setup from source in prepare_base.yml that would setup required repos including centos-nfv-openvswitch repo. So seems can also drop delorean.repo and that can be seperate change. But as part of this patch delorean-deps part can be dropped

Reply:
Notice, the `Containerfile.j2` is part of the `edpm_update` role molecule test. This role uses a container to test. So this containerfile sets up all repos for the test container. When the `edpm_update` role test runs it will include 
```  
ansible.builtin.import_role:
    name: osp.edpm.edpm_ovs
    tasks_from: update
```
So that is the reason I added the extro depo, in order for the edpm_ovs update to find ovs rpms.

The, `edpm_ovs` is **a separate role**. There, we test directly on the host and thus we use `repo-setup` tool to help setup required repos for testing.

Reply:
Thanks for sharing that, i didn't noticed it was changing edpm_update role tests